central/sensor/service/pipeline/imageintegrations/pipeline.go

```diff
@@ -72,20 +72,12 @@ func (s *pipelineImpl) Capabilities() []centralsensor.CentralCapability {
 }
 
 func (s *pipelineImpl) Reconcile(ctx context.Context, clusterID string, storeMap *reconciliation.StoreMap) error {
-	if !features.SourcedAutogeneratedIntegrations.Enabled() {
+	if !features.SourcedAutogeneratedIntegrations.Enabled() && !env.AutogenerateGlobalPullSecRegistries.BooleanSetting() {
 		return nil
 	}
 
 	existingIDs := set.NewStringSet()
 
-	conn := connection.FromContext(ctx)
```

Comment:
just curious why this is gone now. Sorry I forgot to ask the first round

Reply:
Attempted to describe the reason in the description:

> The reconciliation logic referenced a sensor capability centralsensor.ScopedImageIntegrations that I could not find being used in the repo's history. I removed the capability check to allow reconciliation when the feature is enabled.